packages/form-footer/src/FormFooter.tsx

```diff
@@ -58,6 +58,16 @@ const buttonStyle = css`
 /**
  * Types
  */
+interface BaseButtonProps {
+  text?: string;
+  onClick?: React.MouseEventHandler<HTMLButtonElement>;
+}
+
+interface BackButtonProps extends BaseButtonProps {
+  leftGlyph?: React.ReactElement | null;
+}
```

Comment:
Should we allow users to fully customize the buttons? The way I read the ticket, the intent is not only to customize the click handler & left glyph, but allow for further customization. 

@bruugey I left a comment in the JIRA ticket, since I'm a bit confused about the AC

Reply:
Responded in JIRA

Reply:
FWIW I think if we're going as far as accepting a prop object, I would prefer a slot API instead. That's the route we took for CanvasHeader, and I'm sure other devs would appreciate the customizability

Reply:
I think the diff here is that we don't want consumers to change the button variants.

Reply:
yeah from my discussion with design, they only want to introduce button icon customization

so then, customization should be restricted to text + click handler + icon